ministryofjustice/correspondence_tool_staff
# Ct 3694 sql injection brakeman

## Description
Fixes issues with possible SQL injection vulnerabilities 

See speadsheet on ticket for explanations.

## Self-review checklist


### Screenshots


### Related JIRA tickets
[CT-3694](https://dsdmoj.atlassian.net/browse/CT-3694)


### Manual testing instructions
- soft delete a case and check that the redirect works and you get a flash notice when you try to accesses it via URL
- filter cases by exemptions and see if it still works
- sync the case report
